metadata.yaml

```diff
@@ -25,5 +25,9 @@ requires:
   database:
     interface: mongodb_client
 
+provides:  
+  fiveg-nrf:
```

Comment:
```suggestion
  fiveg_nrf:
```

Reply:
I think it's like that in the documentation of the lib :)
https://github.com/canonical/sdcore-nrf-operator/blob/main/lib/charms/sdcore_nrf/v0/fiveg_nrf.py#L40

Reply:
Also in the dummy charms and the test files of the lib.
Let me know if you think it's still better to change it anyway.